app/javascript/stylesheets/lesson-content.css

```
@@ -27,6 +27,38 @@
     @apply bg-gray-50 p-4 border-l-4 border-gold dark:bg-gray-700/40 dark:border-gold-700 rounded;
   }
 
+  .lesson-note > h4::before {
+    font-family: "Font Awesome 6 Free";
+    font-weight: 900;
+    font-size: large;
+    content: "\f249";
+    @apply mr-2 text-gold dark:text-gold-700;
+  }
+
+  .lesson-note--extra {
+    @apply border-purple-600 dark:border-purple-400;
```

If we change this to tip or something like that, green would be a more appropriate color.

```suggestion
    @apply border-green-600 dark:border-green-700;
```

---

Yes, I agree lesson-note--tip would be better with a green colour.